frontend/src/components/Elections.tsx

```diff
@@ -101,7 +112,7 @@ const Elections = ({ authSession }) => {
                             <TableCell> View </TableCell>
                         </TableHead>
                         <TableBody>
-                            {data?.elections_as_voter?.map((election: Election) => (
+                            {data?.elections_as_voter?.filter(election => election.state != 'archived').map((election: Election) => (
```

Comment:
nit

we can probably filter this on the backend within the getElections api, but this is fine for now

Reply:
One thing I'd like to do is add the ability to filter, sort, and add pagination to these tables. I'm thinking the part about filtering out archived elections would just be a preset to those filters.